src/utilities/routes.ts

```diff
@@ -21,7 +21,7 @@ export function getRouterRouteForUrl(routes: ResolvedRoute[], url: string): Rout
   const [route] = matches
   const [, search] = url.split('?')
   const query = createRouterRouteQuery(search)
-  const params = getRouteParams(route, url)
+  const params = getRouteParamValues(route, url)
```

Comment:
just renamed the exported function, now `getRouteParam` is the internal function that generically pulls values off either path or query params